src/main/java/me/zikani/labs/pakadali/api/placeholder/validator/impl/PlaceholderValidator.java

```diff
@@ -0,0 +1,26 @@
+package me.zikani.labs.pakadali.api.placeholder.validator.impl;
+
+import static java.lang.Integer.parseInt;
+
+public class PlaceholderValidator {
```

Comment:
Rename this class, shouldn't be named Validator - because it's doing more than that.

Reply:
Thanks for noting that...will work on it